app/services/fixture_parsing_service.rb

```diff
@@ -6,7 +6,7 @@ class FixtureParsingService
   # Takes an oid and a metadata_source and returns a hash of the fixture file associated with that oid and metadata_source
   def self.fixture_file_to_hash(oid, metadata_source)
     fixture_file_folder = Rails.root.join("spec", "fixtures", metadata_source)
-    file_prefix = FixtureParsingService.file_prefix(metadata_source)
+    file_prefix = file_prefix(metadata_source)
```

Comment:
naming a variable exactly what the method is named can be confusing at best, and mis-read by Ruby in some cases

Reply:
(yes, i see that all we did was pull the class prefix off here, but it made the fact that this was gross more obvious ;-)